features/templates.feature

```diff
@@ -736,7 +736,7 @@ Feature: Templates
                 "test": "{{ user.sign_off }}",
                 "test2": "{{ item.something.missing }}",
                 "test3": "{% if something %}",
-                "test4": "{{ now|iso_datetime }}"
+                "test4": "{{ now|add_timedelta(minutes=45)|iso_datetime }}"
```

Comment:
what/how does it actually test it?

Reply:
I think we don't require it now before that test case is failing here on [first commit](https://github.com/superdesk/superdesk-core/pull/2298#issuecomment-973882915), and later you suggested creating two filters so thought we should add another filter also there. is it ok if I remove that **add_timedelta** filter from here?

Reply:
I think it would be good to test the filter, I'm just not sure what are we testing here exactly